assignments/function-conversion.js

```diff
@@ -3,23 +3,39 @@
 // let myFunction = function () {
 // console.log("Function was invoked!");
 // };
-// myFunction();
+
+let myFunction = ()=> {
+    console.log('Function was invoked')
+}
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+var anotherFunction = param => param;
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+
+let add= (param1, param2) =>{
```

Comment:
If you wanted to, you could drop the return 

ex: let add = (x, y) => ( x + y)